### DISCUSSION TITLE
[Off] Contando sobre minha experiência

### DISCUSSION THREAD

**Discussion by brunobnzt** [frontendbr/forum#1270] [Discussão]
Gostaria de contar uma história que está acontecendo comigo, nem que seja apenas para contar como experiencia de vida para vocês.
Eu trabalho em uma concessionária de carros há 4 anos e alguns meses e, desde que entrei aqui, preciso lidar com um site para enviar boletins referente à ordens de serviço para a fábrica, para que a concessionária possa ser reembolsada do valor de peça e da mão de obra do serviço do mecânico.
Acontece que esse site, onde é feito tal boletim e reembolso, foi feito em aproximadamente 1999 e desde então nunca recebeu uma atualização que seja. Há diversos erros de praticidade e segurança infantis, como por exemplo não poder abrir o mesmo site em duas abas ou dois navegadores simultaneamente, ter que obrigatoriamente filtrar a pesquisa que quero fazer, mesmo não sabendo qual filtro colocar, etc. E também alguns erros de segurança, que se eu quisesse, custaria milhares de reais à fábrica por ano.
Eu já informei à mesma sobre os problemas e sugeri melhorias que nunca foram atendidas. Só nunca informei sobre os problemas de segurança, porque sei que eu seria prejudicado, e é aí que entra onde me enfiei agora.
Há uma certa necessidade que nós concessionários dessa marca sofremos diariamente por falta de interesse da fábrica em resolver esse problema de comunicação. Eu lidei com esse problema por uns dois anos e meio, até que criei um sitezinho bem furreca e hospedei no 000webhostapp e divulguei para o pessoal do grupo do Whatsapp (temos um grupo para compartilhar informações que são porcamente mal repassada pela fábrica). Eles amaram demais a solução e desde então sempre usaram tal site para consultar informações, até que ontem, 05/11/2018, a fábrica mandou uma intimação para a concessionária onde trabalho, exigindo que o site esteja fora do ar. Nem se interessaram em aproveitar a ideia ou me consultar para melhorias no site deles. Apenas ordenaram que tirasse o site e acabou. Ontem mesmo meu gerente veio em minha sala me ameaçando de demissão. Dizendo que a concessionária iria falir e etc.

Bom, a lição que ao menos eu tenho com isso é que, essas histórias de empresas valorizando funcionários que criam soluções para aprimorar e agilizar os processos só acontecem em histórias de jornais. No meu caso me fodi bonito e estou um passo de ir pra rua.

**Comment by rodrigooler** [comment]
@plg01 talvez eu seja um pouco duro, mas talvez chegou a hora de você dar um passo pra ir para rua, MAS se valorizar e encontrar uma empresa onde os interesses sejam os mesmos. Muitas empresas infelizmente vivem com os mesmos problemas, falhas por anos e não resolvem, posso te afirmar que outros funcionários como você já informou esses mesmos problemas. Então a melhor coisa que pode acontecer com você é ser demitido pra encontrar uma nova empresa onde seja valorizado. Você fez um ótimo trabalho e por isto essa empresa não merece te ter como funcionário e sim uma outra empresa disposta a estar aberta a novas ideias e melhorias.

**Comment by frankyston** [comment]
Nossa cara que loucura isso, o que eu posso dizer é que no seu caso você conseguiu enxergar um negócio (pois se existe uma necessidade e essa necessidade foi bem aceita) acredito que você possa contornar isso e transformar em um SAAS. Então no meu ponto de vista acredito que você possa encarar isso e quem sabe ficar até melhor que está agora. Mas isso vai da sua força de vontade e zona de conforto. Mas veja pelo lado bom, você tem uma chance de fazer algo maior.

**Comment by evertonthepaula** [comment]
Se você está usando dados abertos disponibiliza o site para a comunidade :D .

**Comment by brunobnzt** [comment]
> Se você está usando dados abertos disponibiliza o site para a comunidade :D .

Assim que me informaram eu fiquei tão puto que saí excluindo tudo, nem cheguei a salvar nada.

**Comment by tcelestino** [comment]
Complicada a situação, mas é bem isso que o @rodrigooler falou, tenta buscar uma nova oportunidade em um ambiente aonde consiga mostrar o seu talento e que seja valorizado como tal.

**Comment by Edvan27** [comment]
Eu estou passando por algo parecido. Assim, eu trabalho no atacado aqui no Recife, e sou repositor dos frios. Eu preciso sempre verificar as validades e passar para a gerência os produtos que irão para a rebaixa na semana. Para automatizar esse processo, penso em fazer um aplicativo onde leia o código de barra, e eu posso colocar uma descrição e a data que vence, e quando estiver perto, o app mandar uma notificação avisando que irá vencer. Mas aquela velha picuinha de ser mal interpretado por querer ajudar e não aparecer persiste em me deixar travado nisso.

**Comment by evertonthepaula** [comment]
> Eu estou passando por algo parecido. Assim, eu trabalho no atacado aqui no Recife, e sou repositor dos frios. Eu preciso sempre verificar as validades e passar para a gerência os produtos que irão para a rebaixa na semana. Para automatizar esse processo, penso em fazer um aplicativo onde leia o código de barra, e eu posso colocar uma descrição e a data que vence, e quando estiver perto, o app mandar uma notificação avisando que irá vencer. Mas aquela velha picuinha de ser mal interpretado por querer ajudar e não aparecer persiste em me deixar travado nisso.

Cara, que se danem os outros, ninguém gosta de você, ninguém vai te ajudar e todo mundo só quer que você seja tão medíocre quanto eles(desculpe mas eu fico puto!). Corre pelo seu, faça o aplicativo, desenvolva suas habilidades e use você. Se a empresa quiser venda.
